translations/ggdm30.js

```diff
@@ -690,9 +687,29 @@ ggdm30 = {
         var tTags = JSON.parse(tObj.tags);
         for (i in tTags)
         {
-          if (tags[tTags[i]]) hoot.logWarn('Unpacking ZI006_MEM, overwriteing ' + i + ' = ' + tags[i] + '  with ' + tTags[i]);
+          if (tags[tTags[i]]) hoot.logDebug('Unpacking ZI006_MEM, overwriteing ' + i + ' = ' + tags[i] + '  with ' + tTags[i]);
           tags[i] = tTags[i];
-        }
+
+          // Now check if this is a synonym etc. If so, remove the other tag.
+          if (i in ggdm30.fcodeLookupOut) // tag -> FCODE table
+          {
+            if (tags[i] in ggdm30.fcodeLookupOut[i])
+            {
```

Comment:
On input, we re-add the saved tags. The problem is we also have the tags that were added due to the translation of the F_CODE. This function looks through the alternate list of F_CODE translations and if it finds a match, it removes the tag for the "standard" translation
E.g.
Export: natural=wood -> EC015 (Forest) & save natural=wood

This is an alternate translation. landuse=forest is the "standard" translation
Import: EC015 (Forest) -> landuse=forest & we also have natural=wood from the saved tags

The function sees natural=wood and removes landuse=forest